doc/source/deploy-guide/prepare-localhost.rst

```diff
@@ -164,33 +164,32 @@ OpenStack CLI’s ``openstack keypair create``) for accessing the
 instances created. Remember the name of this keypair (which appears as
 ``foctodoodle-key`` in the example below)
 
-Set this for all the following scripts:
+Set this for **all** the following scripts in a deployment:
 
 .. code-block:: console
 
-   export OS_CLOUD=engcloud # Assuming you followed the example for clouds.yaml
-   # Your username plus whatever else you would like, will be used for naming
-   # objects you create in the cloud
-   export PREFIX=foctodoodle
+   export OS_CLOUD=engcloud
+   # 'engcloud' is the name in the `clouds.yaml`,
    # Set the name of the keypair you created
    export KEYNAME=foctodoodle-key
-   # Set the name of the subnet you created
-   export INTERNAL_SUBNET=${PREFIX}-subnet
+   # Set the name of the network you will use in the deployment
+   export INTERNAL_NETWORK=foctodoodle-net
```

Comment:
I don't see the advantage of have now multiple variables you need to remember. Now you need to make sure that your exports of `INTERNAL_NETWORK` and `INTERNAL_SUBNET` match. You can point now to a subnet that is not in the network.